QueryBuilder/Clauses/ColumnClause.cs

```diff
@@ -1,7 +1,10 @@
+using System.Diagnostics;
+
 namespace SqlKata
 {
     public abstract class AbstractColumn : AbstractClause
     {
+        public string Alias { get; set; }
```

Comment:
Why not add the alias only for the sub-type which uses it?

Reply:
We use it to implement `SelectAs()` alias support for 'regular' columns, the alternative is to have two sub-types with both an Alias member.